src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -259,6 +259,12 @@ limitations under the License.
     <!-- This attribute should not be enabled on the server side -->
     <attribute side="server" code="0x00FF" define="UNSUPPORTED" type="boolean" writable="true" optional="true">unsupported</attribute>
 
+    <!-- WARNING: This attribute should not be enabled on the server side. It will return an UNSUPPORTED_ATTRIBUTE if we attempt to Access it while having Admin Privileges -->
```

Comment:
so is the idea here just that you're code-generating stuff for the client side so it can request an attribute on the server side that doesn't exist?